cms/djangoapps/bundles/views.py

```diff
@@ -123,3 +159,59 @@ def bundle_block(request, usage_key_str, view_name='student_view'):
     data.update(fragment.to_dict())
 
     return JsonResponse(data)
+
+
+@api_view(['GET'])
+@view_auth_classes(is_authenticated=True)
+def bundle_xblock_handler_url(request, usage_key_str, handler_name):
+    """
+    Get an absolute URL which can be used (without any authentication) to call
+    the given XBlock handler.
+    
+    The URL will expire but is guaranteed to be valid for a minimum of 2 days.
```

Comment:
Is 2 days the expected standard?

Reply:
It's my guess at a reasonable trade off between "what if the user left their browser open overnight and tried to continue the next day" (should work) vs. "for security, tokens should last too long"

Reply:
Added this to the docstring.